doc/user/bgp.rst

```diff
@@ -3003,7 +3003,9 @@ address-family:
 
    Specifies the route-target list to be attached to a route (export) or the
    route-target list to match against (import) when exporting/importing between
-   the current unicast VRF and VPN.
+   the current unicast VRF and VPN. The `rt vpn export RTLIST` command is not
+   mandatory and can be replaced or completed by the `set extcommunity rt`
+   command in the route-map attached with the `route-map vpn export`.
```

Comment:
it would be nice to have more here ... an example of changing the RT when redistiributing?

Reply:
I added an example.

Reply:
In reply to https://github.com/FRRouting/frr/pull/14512#discussion_r1384926717
You are right. I added a case with using extended-comm-list delete option.